rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :student
+  attr_accessor :date
+
+  def initialize(date, book, student)
+    @date = date
+
+    @book = book
+    book.rentals << self
+
+    @student = student
+    student.rentals << self
```

Comment:
- [ ] please here The rental is adding itself to the @person the 'belongs-to' association and not @student
```ruby
    @person = person
    person.rentals << self
```